src/ViewRenderer.php

```diff
@@ -111,8 +100,32 @@ public static function renderCrawlCache() : void {
             die( 'Forbidden' );
         }
 
-        $view = [];
-        $view['urls'] = CrawlCache::getURLs();
+        if ( ! empty( $_GET['action'] ) && ! empty( $_GET['id'] ) && is_array( $_GET['id'] ) ) {
```

Comment:
I don't have it in this repo's PHPstan/phpcs rules yet, but we'll have checks soon to use `filter_input()` instead of directly accessing `_GET` and `_POST`. So using https://www.php.net/manual/en/function.filter-input.php should allow to remove all the `empty()` calls here.

Reply:
(not asking you to change all references like this if it's not being caught by PHPStan WP rules now. more of a heads up/suggestion)